board-viewer.html

```diff
@@ -747,14 +747,20 @@
           }
 
           entry.name = category.name;
+          entry.href = category.name.toLowerCase().replace(/[^\w ]+/g,'').replace(/ +/g,'-');
 
```

Comment:
My immediate thought on reading the variable name 'href' was -- "If this is an href, it should be escaped to prevent script injection." In looking at how the field is used in board-explorer, it isn't really treated as an href. To prevent confusion, and to better indicate it is a slug of the name, can we rename it "slug"? (We should still pass it through escapeURIComponent just in case it is used by callers as an anchor target.)

@juliocri What do you think?